src/sql/base/browser/ui/radioButton/radioButton.ts

```diff
@@ -109,4 +109,11 @@ export class RadioButton extends Widget {
 	public blur(): void {
 		this.inputElement.blur();
 	}
+
+	private propogatedSetCheck(): void {
```

Comment:
`propagatedSetCheck`

But also I'm not sure this is a clear enough name. Maybe something like `updateInternalState` instead...